packages/node/src/indexer/fetch.service.ts

```diff
@@ -96,7 +96,8 @@ export class FetchService implements OnApplicationShutdown {
     private apiService: ApiService,
     private nodeConfig: NodeConfig,
     @Inject('ISubqueryProject') private project: SubqueryProject,
-    @Inject('IBlockDispatcher') private blockDispatcher: ISubstrateBlockDispatcher,
+    @Inject('IBlockDispatcher')
+    private blockDispatcher: ISubstrateBlockDispatcher,
```

Comment:
This been move to a new line, is this intensional?

Reply:
This would be prettier/eslint